aframe/src/component-anchor.js

```diff
@@ -34,6 +34,12 @@ AFRAME.registerComponent('arjs-anchor', {
 			type: 'number',
 			default: 0.6,
 		},
+
```

Comment:
(thanks a lot for your contributions @nikolaymihaylov, im making comments here but you dont have to do it. i can do the modification myself after the merge)

this seems harmless to emit the event. it is a rare event so no performance impact. we can remove this options for the sake of simplicity